geometry_msgs/Pose marker_pose
sensor_msgs/PointCloud2 cloud
---
sensor_msgs/PointCloud2 cloud
